x/mongo/driver/topology/connection_test.go

```diff
@@ -926,6 +928,23 @@ func TestConnection(t *testing.T) {
 	})
 }
 
+func BenchmarkConnection(b *testing.B) {
+	b.Run("CompressWireMessage CompressorNoOp", func(b *testing.B) {
+		buf := make([]byte, 256)
+		_, err := rand.Read(buf)
```

Comment:
[nit] If we fill this buffer randomly on every call to `BenchmarkConnection` won't this have inconsistent results? I forget how this `rand` is seeded, though.

Reply:
Mostly, this benchmark is for `CompressorNoOp` only, so its content does not matter very much. The actual benchmark starts at l939 underneath.